src/test/java/api/ItemsByInstanceResourceTest.java

```diff
@@ -77,6 +79,39 @@ void canGetInstanceById() {
       hasJsonPath("tenantId", is(TENANT_ID_UNIVERSITY)))));
   }
 
+  @Test
+  void canGetEmptyResult() {
+    UUID instanceId = instancesFixture.basedUponDunkirk().getId();
+
+    // create item in tenant "college"
+    setTempTenantId(TENANT_ID_COLLEGE);
+    IndividualResource collegeLocation = locationsFixture.mainFloor();
+    IndividualResource collegeHoldings = holdingsFixture.defaultWithHoldings(instanceId);
+    IndividualResource collegeItem = itemsFixture.createItemWithHoldingsAndLocation(
+      collegeHoldings.getId(), collegeLocation.getId());
+    clearTempTenantId();
+
+    // create item in tenant "university"
+    setTempTenantId(TENANT_ID_UNIVERSITY);
+    IndividualResource universityLocation = locationsFixture.thirdFloor();
+    IndividualResource universityHoldings = holdingsFixture.defaultWithHoldings(instanceId);
+    IndividualResource universityItem = itemsFixture.createItemWithHoldingsAndLocation(
+      universityHoldings.getId(), universityLocation.getId());
+    clearTempTenantId();
+
+    // make sure neither item exists in current tenant
+    assertThat(itemsFixture.getById(collegeItem.getId()).getResponse().getStatusCode(),
+      is(HTTP_NOT_FOUND.toInt()));
+    assertThat(itemsFixture.getById(universityItem.getId()).getResponse().getStatusCode(),
+      is(HTTP_NOT_FOUND.toInt()));
+
+    ResourceClient.forSearchClient().replace(instanceId, new JsonObject());
```

Comment:
I don't think you need to do all this. You can just call search API with random instance id.

Reply:
makes, sense, thanks for noticing it